app/controllers/renewal/payments_controller.rb

```diff
@@ -17,7 +17,7 @@ def create
         return
       end
 
-      result = checkout.checkout_url(amount: @form.amount, email: @member.email, return_to: callback_renewal_payments_url, member_id: @member.id)
+      result = checkout.checkout_url(amount: @form.amount, email: @member.email, return_to: callback_renewal_payments_url, member_id: @member.id, date: Date.today)
```

Comment:
This a fairly minor thing, but in Rails `Date.today` isn't timezone aware. I think we want to use `Date.current` instead. [This article has a list of does and don'ts](https://thoughtbot.com/blog/its-about-time-zones#do-use) and I usually use it as a cheat sheet because I can't remember the specifics usually.

Reply:
Ah whoops, thanks for catching that and the article!